packages/ods-react/src/components/toggle/src/components/toggleLabels/togglelabels.module.scss

```diff
@@ -0,0 +1,49 @@
+@use '../../../../../style/state';
+
+@layer ods-atoms {
+  .toggle__control__labels {
```

Comment:
you're on your own component there, so no parent context, root name should be component name: `.toggle-labels`